components/DeleteNodeDialog.tsx

```diff
@@ -73,10 +75,11 @@ export function DeleteNodeDialog(props: {
       open
       header={header}
       onClose={handleClose}
-      onConfirm={handleDelete}
+      onConfirm={(_, includeChildren) => handleDelete(includeChildren)}
```

Comment:
Why is not `id` passed as an argument?

Reply:
Because we only need the second argument so that we can pass it to handleDelete. "_" is passed because passing id would give us a ts warning